openmc/capi/__init__.py

```diff
@@ -86,6 +86,20 @@ def _error_handler(err, func, args):
     elif err == _error_code('e_tally_invalid_id'):
         raise KeyError("No tally exists with ID={}.".format(args[0]))
 
+    elif err == _error_code('e_invalid_size'):
+        raise MemoryError("Array size mismatch with memory allocated.")
+
+    elif err == _error_code('e_invalid_fill'):
+        raise GeometryError("Invalid FILL for cell with ID={}.".format(args[0]))
```

Comment:
It's not really that the fill is "invalid", per se, but that the cell has to be filled with a material. Maybe change this error to `E_CELL_NO_MATERIAL`. Also note that `args[0]` is actually the index, not the ID, so maybe just keep it general and don't mention which cell. Perhaps the error message could just be something like "Operation on cell requires that it be filled with a material."